server.js

```diff
@@ -724,6 +724,50 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// AppVeyor test status integration.
+camp.route(/^\/appveyor\/tests\/([^\/]+\/[^\/]+)(?:\/(.+))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var repo = match[1];  // eg, `gruntjs/grunt`.
+  var branch = match[2];
+  var format = match[3];
+  var apiUrl = 'https://ci.appveyor.com/api/projects/' + repo;
+  if (branch != null) {
+    apiUrl += '/branch/' + branch;
+  }
+  var badgeData = getBadgeData('tests', data);
+  badgeData.logo = badgeData.logo || logos['appveyor'];
+  request(apiUrl, { headers: { 'Accept': 'application/json' } }, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+    }
+    try {
+      var data = JSON.parse(buffer);
+      var testsTotal = data.build.jobs.reduce(function(currentValue, job) { return currentValue + job.testsCount; }, 0);
+      var testsPassed = data.build.jobs.reduce(function(currentValue, job) { return currentValue + job.passedTestsCount; }, 0);
+      var testsFailed = data.build.jobs.reduce(function(currentValue, job) { return currentValue + job.failedTestsCount; }, 0);
+
+      if (testsPassed == testsTotal) {
+        badgeData.colorscheme = 'brightgreen';
+      } else if (testsFailed == 0 ) {
+        badgeData.colorscheme = 'green';
+      } else if (testsPassed == 0 ) {
+        badgeData.colorscheme = 'red';
+      } else{
+        badgeData.colorscheme = 'orange';
+      }
+
+      badgeData.text[1] = '  ' + testsPassed + ' / ' + testsFailed + ' / ' + testsTotal + '  ';
```

Comment:
I had to add spaces there otherwise the numbers were overlapping the text in the first block. 
The spaces in between the numbers add readability